packages/gatsby/src/commands/develop-process.ts

```diff
@@ -379,9 +380,24 @@ async function startServer(program: IDevelopArgs): Promise<IServer> {
 
 interface IDevelopArgs extends IProgram {
   graphqlTracing: boolean
+  debugInfo: IDebugInfo | null
+}
+
+const openDebuggerPort = (debugInfo: IDebugInfo): void => {
+  if (debugInfo.break) {
+    inspector.open(debugInfo.port, undefined, true)
+    // eslint-disable-next-line no-debugger
+    debugger
```

Comment:
Is this actually needed? wouldn't 3rd argument to `inspector.open` do what `inspect-brk` does?

Reply:
Yep, else it doesn't actually stop at the next line.

Found some others that are following the same pattern as well: https://github.com/avajs/ava/blob/master/lib/worker/subprocess.js#L206-L211